src/FeedbackBot/Controllers/HomeController.cs

```diff
@@ -89,6 +89,32 @@ public IActionResult Index()
             return RedirectToAction("app", "home", new { appName });
         }
 
+        [HttpPost("search")]
+        public async Task<ActionResult> Search(string searchInput, string appName)
```

Comment:
@cszhu no problems leaving it but usually i'd do a GET instead of a POST for these kinds of methods, since they are repeatable and don't change data (aka idempotent).